runtime/criusupport/j9criu.tdf

```diff
@@ -32,3 +32,8 @@ TraceAssert=Assert_CRIU_mustNotHaveVMAccess noEnv Overhead=1 Level=1 Assert="0 =
 
 TraceException=Trc_CRIU_getNativeString_getStringSizeFail Overhead=1 Level=1 Template="Failed to get Java string size: mutf8String=%s mutf8StringSize=%zu"
 TraceException=Trc_CRIU_getNativeString_convertFail Overhead=1 Level=1 Template="Failed to convert Java string: mutf8String=%s mutf8StringSize=%zu requiredConvertedStringSize=%zd"
+
+TraceEvent=Trc_CRIU_prepare_internal_hooks Overhead=1 Level=5 Template="Prepare for CRIU internal hooks"
```

Comment:
these can all be level2

Reply:
This one is replaced by `runInternalJVMCriuHooks()` entry point which is changed to level 2 along w/ couple of other trace points.